CHANGELOG.md

```diff
@@ -244,3 +244,7 @@ Fixes a bug in icon size for inline `<ChannelValue>` components.
 ## v0.0.1 (May 24, 2019)
 
 Initial beta release
+
+### Updated 
+
+Updated react component library to 5.3.2
```

Comment:
You only add the changelog at the top. And it should be more descriptive. See how Angular component library is doing it:

https://github.com/pxblue/angular-component-library/blob/dev/CHANGELOG.md

Reply:
@huayunh Thanks for the feedback. Updated the PR.